stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -1315,10 +1307,11 @@ public func _setDownCast<BaseValue, DerivedValue>(_ source: Set<BaseValue>)
   _sanityCheck(_isClassOrObjCExistential(DerivedValue.self))
 
   switch source._variantStorage {
-  case _VariantSetStorage.native(let nativeOwner):
+  case _VariantSetStorage.native(let storage):
+    let bridgingStorage = storage.bridgingStorage()
     return Set(
     _immutableCocoaSet:
-    unsafeBitCast(nativeOwner, to: _NSSet.self))
+    unsafeBitCast(bridgingStorage, to: _NSSet.self))
```

Comment:
While you are changing this code, would you mind fixing the indentation?  `_immutableCocoaSet:` should be indented 2 spaces because it is a line continuation, and if `unsafeBitCast` does not fit on the previous line, it should have 2 spaces more than `_immutableCocoaSet:`.